benefits/core/templates/core/includes/media-list.html

```diff
@@ -0,0 +1,18 @@
+<ul class="media-list">
+    {% for item in media %}
+        <li class="media">
+            <div class="media-line">{% include "core/includes/icon.html" with icon=item.icon %}</div>
+            <div class="media-body">
+                <h3 class="media-body--heading">{{ item.heading }}</h3>
+                <div class="media-body--details">
+                    <p>{{ item.details }}</p>
+                    <div class="media-body--items">
```

Comment:
Wrap in `{% if %}` to avoid rendering `media-body--items` if no items

Reply:
Done in https://github.com/cal-itp/benefits/pull/967/commits/1ef48ce4355e00135b41155f159c0e3efcd235fe for media-list include.

![image](https://user-images.githubusercontent.com/25497886/193697711-09a8cb97-cece-4830-89c7-48f7bcb9b724.png)